lib/rules/property-units.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+var helpers = require('../helpers');
+
+module.exports = {
+  'name': 'property-units',
+  'defaults': {
+    'unitsAllowedForProperties': {},
+    'globallyAllowedUnits': []
+  },
+  'detect': function (ast, parser) {
+    var result = [];
+    var globallyAllowedUnits = parser.options.globallyAllowedUnits;
```

Comment:
Mixin var declaration styles here and [here](https://github.com/sasstools/sass-lint/pull/387/files#diff-ecba41c08d8d6d5f4222fcbe86d47722R18), if we stick with just trailing commas in this instance.